.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: stratisd release
+
+# yamllint disable-line rule:truthy
+on:
+  release:
+    types:
+      - created
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  allowed_failures:
```

Comment:
We don't want this one to be an allowed failure, so we should give the job a different title. Might as well drop continue-on-error setting as well. With those changes, should be ready to go in.